src/Redux/actions.tsx

```diff
@@ -123,6 +124,173 @@ export const listAssets = (params: object) =>
+  fireRequest("listAssetTransaction", [], params);
+export const getAssetTransaction = (id: string) =>
+  fireRequest("getAssetTransaction", [], {}, { id });
+
+export const listAssetService = (params: object, asset_external_id: string) =>
+  fireRequest("listAssetService", [], params, { asset_external_id });
+export const getAssetService = (
+  params: object,
+  asset_external_id: string,
+  external_id: string
+) =>
+  fireRequest("getAssetService", [], params, {
+    asset_external_id,
+    external_id,
+  });
+export const updateAssetService = (
+  asset_external_id: string,
+  external_id: string,
+  params: object
+) =>
+  fireRequest("updateAssetService", [], params, {
+    asset_external_id,
+    external_id,
+  });
+
+// ABDM related
+export const generateAadhaarOtp = (aadhaar: string) =>
+  fireRequest("generateAadhaarOtp", [], { aadhaar });
+
+export const resentAadhaarOtp = (txnId: string) =>
+  fireRequest("resendAadhaarOtp", [], { txnId });
+
+export const verifyAadhaarOtp = (txnId: string, otp: string) =>
+  fireRequest("verifyAadhaarOtp", [], { txnId, otp });
+
+export const generateMobileOtp = (txnId: string, mobile: string) =>
+  fireRequest("generateMobileOtp", [], { txnId, mobile });
+
+export const checkAndGenerateMobileOtp = (txnId: string, mobile: string) =>
+  fireRequest("checkAndGenerateMobileOtp", [], { txnId, mobile });
+
+export const verifyMobileOtp = (txnId: string, otp: string) =>
+  fireRequest("verifyMobileOtp", [], { txnId, otp });
+
+export const createHealthId = (data: ICreateHealthIdRequest) =>
+  fireRequest("createHealthId", [], data);
+
+export const searchByHealthId = (healthId: string) =>
+  fireRequest("searchByHealthId", [], { healthId });
+
+export const initiateAbdmAuthentication = (
+  authMethod: string,
+  healthid: string
+) => fireRequest("initiateAbdmAuthentication", [], { authMethod, healthid });
+
+export const confirmWithAadhaarOtp = (
+  txnId: string,
+  otp: string,
+  patientId?: string
+) => fireRequest("confirmWithAadhaarOtp", [], { txnId, otp, patientId });
+
+export const confirmWithMobileOtp = (
+  txnId: string,
+  otp: string,
+  patientId?: string
+) => fireRequest("confirmWithMobileOtp", [], { txnId, otp, patientId });
+
+export const linkViaQR = (abha_details: any, patientId?: string) => {
+  return fireRequest("linkViaQR", [], {
+    ...abha_details,
+    patientId,
+  });
+};
+
+export const linkCareContext = (
+  consultationId: string,
+  data: { name?: string; gender?: "M" | "F" | "O"; dob?: string }
+) => {
+  return fireRequest("linkCareContext", [], {
+    consultation: consultationId,
+    ...data,
+  });
+};
+
+export const getAbhaCard = (patient: string, type: "pdf" | "png") => {
+  return fireRequest("getAbhaCard", [], {
+    patient,
+    type,
+  });
+};
+
+export const getHealthInformation = (artefactId: string) => {
+  return fireRequest("getHealthInformation", [], { artefactId });
+};
+
+export const healthFacilityActions = {
+  list: (params: object) => {
```

Comment:
Were these added by accident / incorrect merge conflict resolution?

There seem to be 0 references to these redux actions since it was already migrated.


<img width="1024" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/6ce86d53-1154-4bed-bad0-2784b1bea779">